docs/news.d/try_compile.rst

```diff
@@ -2,10 +2,12 @@
 
 .. news-start-section: Platform Support and Dependencies
 .. news-start-section: CMake
+.. news-start-section: Building OpenDDS with CMake
 - The :ref:`shapes-demo` will now be built when :ref:`cmake-building` and :cmake:var:`OPENDDS_QT` is set to ``TRUE``.
+- Invalid :cmake:var:`OPENDDS_ACE_TAO_KIND` values will now be explictly rejected.
```

Comment:
Is this user-facing?

Reply:
[Yes](https://opendds.readthedocs.io/en/master/devguide/building/index.html#cmake-var-OPENDDS_ACE_TAO_KIND), it's how the user would choose what ACE/TAO to get. Whether or not this is news worthy could be debated though.

Reply:
Right, I was thinking that it doesn't need to be in the NEWS